internal/workos/http.go

```diff
@@ -25,23 +27,39 @@ func TryGetHTTPError(r *http.Response) error {
 	} else {
 		msg = string(body)
 	}
+	if e := gjson.GetBytes(body, "error").Str; e != "" {
```

Comment:
We removed `gjson` as a dependency in #115, so we'll have to adjust our approach.

Reply:
@maxdeviant I've been working on this but can't seem to use your logic to accomplish the same behavior I was getting through `gjson.GetBytes(body, "error")` or `gjson.GetBytes(body, "error_description")`

If you have any pointers, I'd appreciate it.

Reply:
For example, just attempting to use the same logic and extract error description doesn't seem to work:
```
func getJsonErrorDescription(b []byte) string {
	var response struct{ ErrorDescription string }

	if err := json.Unmarshal(b, &response); err != nil {
		return ""
	}

	return response.ErrorDescription
}
```